<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Add</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionChoose"/>
   </widget>
   <addaction name="menuAdd"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionChoose">
   <property name="text">
    <string>Choose</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
